_health-care/_js/components/form-elements/GrowableTable.jsx

```diff
@@ -28,37 +46,77 @@ class GrowableTable extends React.Component {
     this.props.onRowsUpdate(rows);
   }
 
+  // TODO: change view to not be dependent on index number, this is creating UI problems
   render() {
+    let reactKey = 0;
```

Comment:
I found that this method of generating a key is an anit-pattern as is it will cause unnecessary DOM thrashing.

Instead, we should be storing the individual rowElements instead of create new ones per render.